Misc/NEWS.d/next/Core and Builtins/2017-10-17-16-46-49.bpo-25711.gfGaqL.rst

```diff
@@ -0,0 +1 @@
+zipimport has been rewritten in pure Python.
```

Comment:
A separate news entry for the DeprecationWarning -> TypeError change for creating zip importers directly from bytes paths would probably make sense. (Or at least mentioning it in this one)